AXIOM_Remote_Firmware_Visualizer/VirtualUI.cpp

```diff
@@ -583,13 +586,19 @@ void VirtualUI::RenderUI(Button& button, int8_t& knobValue, bool& debugOverlayEn
     ImGui::SetCursorPos(ImVec2(337, 119));
     ShowZoomTooltip();
 
-    ImGui::SetCursorPos(ImVec2(50, 390));
+    ImGui::SetCursorPos(ImVec2(50, 375));
     ImGui::ToggleButton("debug_overlay_switch", "Debug overlay", &debugOverlayEnabled);
 
     ToggleLCDContrast(toggleContrast);
-    ImGui::SetCursorPos(ImVec2(50, 430));
+    ImGui::SetCursorPos(ImVec2(50, 410));
     ImGui::ToggleButton("toggle_contrast_switch", "Simulate LCD", &toggleContrast);
 
+    ImGui::SetCursorPos(ImVec2(50, 445));
+    if (ImGui::Button("Take Screenshot"))
+    {
+        _ScreenshotHandler();
```

Comment:
See the comment above. What happens if the handler is not given, e.g. nullptr or so?

Reply:
If no argument is given, the default argument (an empty lambda function) will be used, which means the button will do nothing.
If a nullptr is sent instead, it will currently fail silently. 
It might be a good idea to add an additional check: `if (ImGui::Button("Take Screenshot") && _screenshotHandler)`